### PR TITLE
Fix some Typos and capitalization

### PR DESCRIPTION
**Pull request by TaserTheFox**
Renamed the Red stuff, added a possibility to remove the tree sapling recipe. 

Also, it seems that "orange ingot is more involved" and "cage can capture" lines are never called?

pls tell me how to remove recipes

### CODE REVIEW

File: kubejs/assets/minecraft/lang/en_us.json [hunk 1]
```diff
@@ -29,6 +33,8 @@
     "block.minecraft.gold_block": "Block of Yellow",
     "block.minecraft.iron_block": "Block of White",
     "block.minecraft.diamond_ore": "Light Blue Ore",
+    "item.minecraft.redstone_dust": "Red Dust",
```
Comment (Yarden-zamir) [inline kubejs/assets/minecraft/lang/en_us.json:36]: I do want redstone to stay redstone, it has a personality id like to keep

**Review by Yarden-zamir — CHANGES_REQUESTED** [review]
This is awesome, i do want redstone to stay though so lets revert that part before merging

File: kubejs/assets/minecraft/lang/en_us.json (hunk 1 above)
Comment (TaserTheFox) [reply kubejs/assets/minecraft/lang/en_us.json]: changed!

**Comment by TaserTheFox** [comment]
Hey i accidentally added a new feature into this request, ill make it into a new pr next time qwq

anyway please re-review, I removed tree fert.

**Review by Yarden-zamir — APPROVED** [review]

**Review by Yarden-zamir — APPROVED** [review]

**Comment by Yarden-zamir** [comment]
need to fix the conflict and merge

**Comment by Yarden-zamir** [comment]
I'll do it when I can as I need computer for that but if you do it before me I'll merge

**Comment by TaserTheFox** [comment]
Fixed conflict, ready to be merged

**Review by Yarden-zamir — APPROVED** [review]

**Comment by Yarden-zamir** [comment]
merged, also note that when doing fixes you can use `fix: fix name` in commit message for it to show in the chaneloge

**Pull request merged**